.github/workflows/tests.yml

```diff
@@ -20,7 +20,8 @@ jobs:
           node-version: '12.x'
       - uses: bahmutov/npm-install@v1
       - run: npm run build
-      - run: npm test
+      - run: sudo apt-get install xvfb
+      - run: xvfb-run --auto-servernum npm test
```

Comment:
```suggestion
      - run: sudo apt-get install xvfb
      # use xvfb to run the tests with a pseudo-display attached
      # so that the browsers started by karma think there is a screen
      - run: xvfb-run --auto-servernum npm test
```